dali/sasha/saserver.cpp

```diff
@@ -469,8 +498,11 @@ int main(int argc, const char* argv[])
                         if (!stopped)
                         {
                             stopped = true;
-                            IInterCommunicator &comm=queryWorldCommunicator();
-                            comm.cancel(NULL, MPTAG_SASHA_REQUEST);
+                            if (isDaliClient)
+                            {
+                                IInterCommunicator &comm = queryWorldCommunicator();
+                                comm.cancel(NULL, MPTAG_SASHA_REQUEST);
+                            }
                         }
                     }
                 } *stopThread = new CStopThread;
```

Comment:
unconditionally creating CStopThread here, but conditionally deleting it on line 521.
If not a Dali client, then doesn't need this thread.
It doesn't need to signal the stopSem either (doesn't need the stopSem at all, but harmless).